### DISCUSSION TITLE
Version php incompatible suite à mise à jour

### DISCUSSION THREAD

**Discussion by Bedeau69** [FreshRSS/FreshRSS#7141] [Q&A] [answered]
Bonjour à tous !

J'ai installé hier la mise à jour qui nécessite php 8.1. Sauf que mon (vieux) Synology me permet uniquement une version 8.0.

1ère question : la mise mise à jour automatique n'aurait-elle pas dû m'avertir de la non compatibilité avec php 8.0 ? Peut-être que le message était bien présent mais j'avoue que depuis le temps, je clique rapidement vu que toutes les maj passent sans aucun problème...

2ème question : il me semble qu'il y a une sauvegarde qui est réalisée juste avant la mise à jour. Mais je ne trouve pas beaucoup de chose sur l'endroit où elle se trouve, le nom de l'archive et comment on fait pour restaurer... Si vous avez une piste de recherche...

J'ai essayé de remplacer le répertoire ./data/ par celui qui s'appelle ./data.back/ mais cela n'a pas fonctionné.
(désolé de mes questions peut-être bêtes :-)

Merci
Vincent

**Comment by Bedeau69** [comment]
(à priori, je n'avais pas ouvert ma question au bon endroit, sorry, pas très à l'aise avec Github !)

**Comment by Alkarex** [comment, marked as answer]
> 1ère question : la mise mise à jour automatique n'aurait-elle pas dû m'avertir de la non compatibilité avec php 8.0 ? Peut-être que le message était bien présent mais j'avoue que depuis le temps, je clique rapidement vu que toutes les maj passent sans aucun problème...

Oui. Tout le système de mise à jour est en attente d'une aide depuis fort longtemps https://github.com/FreshRSS/FreshRSS/pull/1760

Des avertissements avaient été donnés dans cette version ainsi que les précédentes
* https://github.com/FreshRSS/FreshRSS/releases/tag/1.25.0
* https://github.com/FreshRSS/FreshRSS/releases/tag/1.24.3

> 2ème question

Je vous suggère de juste redéployer (après une sauvegarde) l'[ancienne version](https://github.com/FreshRSS/FreshRSS/releases/tag/1.24.3) par dessus vos fichiers actuels, et ça devrait aller

Vous pourriez aussi regarder si votre Synology supporte Docker, ce qui serait une autre possibilité

**Comment by Bedeau69** [comment]
Merci pour ce retour rapide ! 

J'ai remonté une sauvegarde de mes fichiers et des services Web et ça fonctionne. 

Va falloir réfléchir bien fort à la suite et se rappeler de bien tout lire avant de mettre à jour des choses, petite pique de rappel !

Joyeuses fêtes à tous :-)

**Comment by math-GH** [comment]
This inspired me to improve the web update script: https://github.com/FreshRSS/update.freshrss.org/pull/20
